### PR TITLE
[MOB-4939] Update react-example app and remove variable from env file

### PR DESCRIPTION
**Pull request by pauljung14**
## JIRA Ticket(s) if any

* [MOB-4939](https://iterable.atlassian.net/browse/MOB-4939)

## Description

Some QoL improvements to the react-example app.
Removing support for GETMESSAGES_PATH env variable, because we actually don't want user to change this path 😅 

## Test Steps
Unit tests all pass.
Cannot change getMessages request path via .env file.

### CODE REVIEW

File: react-example/src/views/InApp.tsx [hunk 1]
```diff
@@ -59,11 +59,11 @@ export const InApp: FC<{}> = () => {
       .then((response) => {
         setRawMessageCount(response.data.inAppMessages.length);
         setIsGettingMessagesRaw(false);
-        setGetMessagesResponse(JSON.stringify(response.data));
+        setGetMessagesResponse(JSON.stringify(response.data, null, 2));
```
Comment (pauljung14) [inline react-example/src/views/InApp.tsx:62]: auto-formats the json response displayed in the sample app

File: react-example/src/views/InApp.tsx [hunk 2]
```diff
@@ -33,7 +33,7 @@ const { request, pauseMessageStream, resumeMessageStream } = getInAppMessages(
     count: 20,
     packageName: 'my-website',
     closeButton: {},
-    displayInterval: 10000
+    displayInterval: 1000
```
Comment (pauljung14) [inline react-example/src/views/InApp.tsx:36]: reducing wait time interval, 10 seconds is too long for testing purposes of this app, and user can always change it if desired

File: react-example/src/views/InApp.tsx [hunk 3]
```diff
@@ -146,6 +131,21 @@ export const InApp: FC<{}> = () => {
...
+          data-qa-get-messages-raw
+        >
+          {typeof rawMessageCount === 'number'
+            ? `Retrieved ${rawMessageCount} messages (try again)`
+            : 'Get Messages (do not auto-display)'}
+        </GetMessagesRawButton>
+        <Response data-qa-get-messages-raw-response>
+          {getMessagesResponse}
+        </Response>
+      </EndpointWrapper>
```
Comment (pauljung14) [inline react-example/src/views/InApp.tsx:148]: Just moving this section below the auto-display section

**Review by mprew97 — APPROVED** [review]
👍 👍

**Pull request merged**